### ISSUE TITLE
Discovery for 526 Start Identification Error 

### ISSUE THREAD

**Issue by Nyanteea** [department-of-veterans-affairs/va.gov-team#56676]
## High Level User Story

Survey feedback has revealed that a number of 526 users receive errors a long the lines of "We need more information from you before you can file a Benefits Delivery at Discharge claim with VA Form 21-526EZ. Please call Veterans Benefits Assistance at 800-827-1000 (TTY: 711), Monday through Friday, 8:00 a.m. to 9:00 p.m. ET to update your account"

The message seems to vary depending on what ID is missing. We have identified one message specifically for BIRLS, IDIPI but we need to know what the others are in order to make content changes. 

[Link to Slack thread for more context ](https://dsva.slack.com/archives/CSFV4QTKN/p1680789142637669?thread_ts=1680720909.972049&cid=CSFV4QTKN)

## Need to Know
- Need to understand how many error messages there are and which pieces of information are they tied to? e.g. One message specifically mentions a missing BIRLS, IDIPI credential, do other errors exist that tell the user exactly what is missing? 
- Does a different error message display depending whether a SSN, DoB, EDIPI, BIRLS ID or Participant ID is missing or incorrect 
- Is the updated Contact Center number relevant for all issues concerning account IDs/credentials or just IDIPI/BIRLS?

## Goals
_What outcome(s) do we want to see?_ 
- [ ] Meet vsp-identity to discuss issue further - Joe Niquette, Kyle, Robin to understand problem better - requires engineering participation
  - [ ] Document what error msgs/flows would show up for this issue 
  - [ ] Identify which alerts require phone number update for ticket #57110
- [ ] Check Google Analytics for "va-alert"
- [ ] Determine possible engineering solutions
- [ ] Determine possible content updates to inform veteran what to do if they see this issue
- [x] Discuss with Contact Center to see what potential solutions/blockers there are - A new ticket has been created for this

## Issues to create within this Epic

```
- Create research plan
- Recruit for research sessions/Schedule research sessions
- Create conversation guide
- Conduct Discovery Research Sessions (at a minimum, the team lead, PM, designer, content person should join these sessions).
- Discovery Synthesis - should be conducted as a team, led by designer.
- Discovery Research Readout (at minimum, it is expected that the designer should do a readout at design meetings & for their team. If stakeholder readouts are needed, the team lead/PM are responsible for scheduling those. Designers should also plan to do research readouts at the end-of-sprint demo.)
```

## How to configure this issue
- [ ] **Labeled with `discovery` and `research` and `team-name`**

**Comment by Nyanteea** [comment]
@ahay-agile6 Hi Aurora -- do you already have a ticket addressing this error that should be linked here?

**Comment by Nyanteea** [comment]
@lydiahooper

**Comment by Nyanteea** [comment]
Contact Center team[ created a ticket](https://app.zenhub.com/workspaces/contact-center-62cdd9546ec1530018209672/issues/gh/department-of-veterans-affairs/va.gov-team/49356) with the updates phone number and content which has been waiting for our team to integrate the changes

**Comment by RakshindaAslam** [comment]
[Update phone number on 526-EZ alert #49356](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/57110) created on 526 backlog

**Comment by Nyanteea** [comment]
a way to repro this in the dev environment is as such:
sign in to dev.va.gov as id.me user vets.gov.user+226@gmail.com
browse to https://dev.va.gov/disability/file-disability-claim-form-21-526ez/introduction
if it puts you back to "[...]/start", click on "If you know VA Form 21-526EZ is right, apply now"
the aforementioned error shows up
a user that this does not show up for is vets.gov.user+228@gmail.com

**Comment by sethdarragile6** [comment]
FYI, possibly related [thread](https://dsva.slack.com/archives/C04KW0B46N5/p1686148130634419)

**Comment by NB28VT** [comment]
_**(In progress, posting for visibility)**_

## Update

The first piece of information needed here is a mapping of the different error messages and which states on the front end cause them to be displayed. Yesterday I walked through the code and reasoned this out. As it's a logic flow that is better expressed visually, I am mapping this out in a Mural doc:

[Mural logic mapping](https://app.mural.co/t/coforma8350/m/coforma8350/1687888432172/45e791205c9f0d4517ec3aa5d0d1c5bdf9fead26?sender=u846d384c8bd13e5d451c0747)

Once this is done I will move on to cataloging the sources of this data on the backend to provide a better understanding of the services at play.
